Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
+
+gem "activesupport", "= 7.0.8"
+gem "cocoapods", "= 1.13.0"
```

Comment:
`m`: @brustolin, please add a comment explaining why we need to pin these versions, and why we can't use `gem install cocoapods --no-document -v 1.21.11.